flipperdevices/flipperzero-good-faps
# Picopass: More info view (raw bytes)

# What's new

- "More" off of "info" view with raw bytes

# Verification 

- Save saved picopass card or read one and save it
- Select "Saved"
- Select a saved card
- Select "Info"
- Push right for "More"
- See raw bytes of picopass card
- Scroll up/down
- Hit "Back" to exit

# Checklist (For Reviewer)
